pkg/operator/certrotation/cabundle.go

```diff
@@ -39,17 +40,20 @@ func (c CABundleRotation) ensureConfigMapCABundle(signingCertKeyPair *crypto.CA)
 	if err != nil && !apierrors.IsNotFound(err) {
 		return nil, err
 	}
-	caBundleConfigMap := originalCABundleConfigMap.DeepCopy()
```

Comment:
let's just keep this line. it avoids special cases.

Reply:
that line just moved... why deepcopy when we not found and we don't need to deep copy?

Reply:
> that line just moved... why deepcopy when we not found and we don't need to deep copy?

to avoid an unnecessary branching condition